tests/vcd_disk.py

```diff
@@ -180,5 +180,16 @@ def test_010_get_storage_profile(self):
         print("Storage Profile: " + str(profile))
         assert profile.get('name') == self.config['vcd']['storage_profile']
 
+    def test_111_change_owner_disk(self):
+        logged_in_org = self.client.get_org()
+        org = Org(self.client, resource=logged_in_org)
+        vdc_resource = org.get_vdc(self.config['vcd']['vdc'])
+        user_resource = org.get_user(self.config['vcd']['new_disk_user'])
+        vdc = VDC(self.client, href=vdc_resource.get('href'))
+        disk = vdc.change_disk_owner(self.config['vcd']['disk'], user_resource.get('href'), self.config['vcd']['disk_id'])
```

Comment:
Line too long. Can you run flake8 tests and verify that the line length is not > 80 ?